src/test/ts/part3/Part3Ex3Test.ts

```diff
@@ -0,0 +1,108 @@
+import { Cursors } from '@ephox/agar';
+import { beforeEach, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyDom, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { Traverse } from '@ephox/sugar';
+import { assert } from 'chai';
+import { Editor } from 'tinymce';
+
+describe('Part3Ex3Test', () => {
+  const hook = TinyHooks.bddSetup<Editor>({ height: '100vh' });
+
+  beforeEach(() => {
+    // Fun fact, hook.editor() also works inside of before / beforeEach
+    // callbacks: as long as you declare the hook first.
+
+    const editor = hook.editor();
+    editor.setContent([
+      '<p>Here is a bit of content</p>',
+      '<p>And some <strong>bolded</strong> content</p>',
+      '<p><img src="https://http.cat/301.jpg" alt="A cat picture" /></p>'
+    ].join(''));
+  });
+
+  it('selects content like an editor', () => {
+    const editor = hook.editor();
+
+    /*
+    Now we're going to talk about one of the scarier parts of editor
```

Comment:
```suggestion
    Now we're going to talk about one of the more complicated parts of editor
```
is this better wording maybe? :P